Apps/Medication/src/Delegates/SalesforceDelegate.cs

```diff
@@ -18,33 +18,33 @@ namespace HealthGateway.Medication.Delegates
     using System;
     using System.Collections.Generic;
     using System.Diagnostics;
-    using System.Net;
     using System.Net.Http;
-    using System.Net.Http.Headers;
-    using System.Net.Mime;
-    using System.Text.Json;
     using System.Threading.Tasks;
     using AutoMapper;
     using HealthGateway.Common.AccessManagement.Authentication;
     using HealthGateway.Common.Data.Constants;
     using HealthGateway.Common.Data.ViewModels;
     using HealthGateway.Common.Delegates;
     using HealthGateway.Common.ErrorHandling;
-    using HealthGateway.Common.Services;
+    using HealthGateway.Medication.Api;
     using HealthGateway.Medication.Models;
     using HealthGateway.Medication.Models.Salesforce;
     using Microsoft.Extensions.Configuration;
     using Microsoft.Extensions.Logging;
+    using Refit;
 
     /// <summary>
     /// Salesforce Implementation that retrieves Medication Requests.
     /// </summary>
     public class SalesforceDelegate : IMedicationRequestDelegate
     {
-        private const string SalesforceConfigSectionKey = "Salesforce";
+        /// <summary>
+        /// The key used to lookup Salesforce configuration.
+        /// </summary>
+        public const string SalesforceConfigSectionKey = "Salesforce";
```

Comment:
Since these config section constants no longer need to be accessed inside the delegates, I think we should move them into the appropriate Config model, i.e. HealthGateway.Medication.Models.Salesforce.Config.